package.json

```diff
@@ -31,17 +31,20 @@
         "@mantine/spotlight": "^5.3.0",
         "@tabler/icons": "^1.95.1",
         "@tauri-apps/api": "^1.0.2",
+        "node": "^22.7.0",
```

Comment:
Hi @slpysiren,I encountered the same error as Michael when trying to build the DDT tool (Error: Cannot find module). It seems the issue is due to the addition of the Node version in package.json. Removing this line resolved the problem for me.

Reply:
Ah good catch, don't remember why I added node as a dependency anyway so removing should fix. Thanks!

Reply:
I can confirm that it is working for me now too 😊